### DISCUSSION TITLE
Flyout Scrollbar

### DISCUSSION THREAD

**Discussion by sachssoft** [AvaloniaUI/Avalonia#12923] [Q&A] [answered]
Hello,

The 'Width' property doesn't work if it is larger than about 500. But the 'Height' works.

How can I disable Flyout's HorizontalScrollbar?

`

		
		<DropDownButton>
			<Path Data="..." />
			<DropDownButton.Flyout>
				<Flyout Placement="BottomEdgeAlignedLeft">
					<Grid ColumnDefinitions="1*, 1*, 1*, 1*"
						  RowDefinitions="Auto, *" Width="600">
						<Grid.Styles>
							<Style Selector="ItemsControl">
								<Setter Property="ItemsPanel">
									<ItemsPanelTemplate>
										<StackPanel Orientation="Vertical" />
									</ItemsPanelTemplate>
								</Setter>
								<Setter Property="ItemTemplate">
									<DataTemplate>
										<Button Content="Cnt"
												CornerRadius="0"
												HorizontalAlignment="Stretch" />
									</DataTemplate>
								</Setter>
							</Style>
						</Grid.Styles></Grid>
				</Flyout>
			</DropDownButton.Flyout>
		</DropDownButton>
`

**Comment by jp2masa** [comment, marked as answer]
I think you have to override the default `FlyoutPresenter`'s `MaxWidth`, either by setting it in a style or overriding this resource:

- https://github.com/AvaloniaUI/Avalonia/blob/25be13361356f7784314fb20b5a76ac462b5e2b8/src/Avalonia.Themes.Fluent/Accents/BaseResources.xaml#L37

**Reply by sachssoft** [reply]
If the theme is custom overridden, MaxWidth can be changed. Actually it worked. Thank you so much

`
<Flyout.FlyoutPresenterTheme>
<ControlTheme TargetType="FlyoutPresenter">
<Setter Property="Template">
								<ControlTemplate>
									<FlyoutPresenter MaxWidth="800"
													 MinHeight="300"
													 CornerRadius="0"
													 Content="{TemplateBinding Content}">
									</FlyoutPresenter>
								</ControlTemplate>
							</Setter>
						</ControlTheme>
					</Flyout.FlyoutPresenterTheme>
`
